src/instructlab/eval/data/mt_bench/reference_answer/gpt-4.jsonl

```diff
@@ -0,0 +1,30 @@
+{"question_id": 101, "answer_id": "TFomieEmmAgdeCkvmuvwbc", "model_id": "gpt-4", "choices": [{"index": 0, "turns": ["If you have just overtaken the second person, your current position is now second place. The person you just overtook is now in third place.", "If you have just overtaken the last person, it means you were previously the second to last person in the race. After overtaking the last person, your position remains the same, which is second to last. The person you just overtook is now in the last place."]}], "tstamp": 1686286924.844282}
```

Comment:
Is it fine that `model_id` is `gpt-4` even though we are using Prometheus2 instead of GPT-4?

Reply:
Yes, the reference answers are provided by gpt-4, the model answers are provided by the model being judged.  And the judgment of how good the model being judged is compared to the gpt-4 answer is provided by prometheus.